cpp/nodes_module/algorithm/nodes.cpp

```diff
@@ -1 +1,24 @@
-#include "nodes.hpp"
\ No newline at end of file
+#include "nodes.hpp"
+
+void Nodes::Link(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory) {
+  mgp::memory = memory;
+  const auto arguments = mgp::List(args);
+  const auto record_factory = mgp::RecordFactory(result);
+  try {
+    mgp::Graph graph = mgp::Graph(memgraph_graph);
+    const mgp::List list_nodes = arguments[0].ValueList();
+    const std::string_view type{arguments[1].ValueString()};
+    const size_t size = list_nodes.Size();
+    if (size < 2) {
```

Comment:
Extract 2 as constant in header file. This is usually considered magic number (number which appears out of nowhere), and general practice is to extract it into constant